packages/fxa-auth-server/lib/senders/emails/partials/location/index.txt

```diff
@@ -0,0 +1,6 @@
+<% if (locals.primaryEmail) { %><%- primaryEmail %><% } %>
+<% if (locals.device) { %><%- device %><% } %>
+<% if (locals.location) { %><%- location %><% } %>
+<% if (locals.ip) { %> user-ip = "IP address: <%- ip %>" <% } %>
+<% if (locals.date) { %><%- date %><% } %>
+<% if (locals.time) { %><%- time %><% } %>
```

Comment:
I tried using ternary operator for conditional rendering something like this: `<%- locals.time ? time : "" %>` but a new line empty line gets added if the variable is not available which looks a little inconsistent in the final plaintext file